dawn/src/dawn/CodeGen/CXXNaive-ico/CXXNaiveCodeGen.cpp

```diff
@@ -628,7 +628,10 @@ std::unique_ptr<TranslationUnit> CXXNaiveIcoCodeGen::generateCode() {
   ppDefines.push_back("#define DAWN_GENERATED 1");
   ppDefines.push_back("#undef DAWN_BACKEND_T");
   ppDefines.push_back("#define DAWN_BACKEND_T CXXNAIVEICO");
+  ppDefines.push_back(
+      "#define GRIDTOOLS_DAWN_NO_INCLUDE"); // Required to not include gridtools from math.hpp
```

Comment:
I get an include error because "math.hpp" includes "storage.hpp", which includes stuff from gridtools (and we don't depend on gridtools in the unstructured tests or in icondusk-e2e) ... apparently `GRIDTOOLS_DAWN_NO_INCLUDE` needs to be set